src/index.js

```diff
@@ -2,11 +2,12 @@ require('@babel/polyfill')
 
 const server = require('./server')
 
-server.init()
-  .then(({ app }) => {
-    console.log('\n\n server started on port 3000')
-  })
-  .catch(err => {
-    console.error(err)
-    process.exit(1)
-  })
+;(async () => {
```

Comment:
Same comment from `scripts/lighthouse.js`

Reply:
@bmullan91 same linter issue ( at least ), I originally put things like that because I had an error (might be a linter one already at that time), because an `await` should be inside an `async function`